playback-service/src/main/java/com/playback/models/VideoMetadata.java

```diff
@@ -0,0 +1,73 @@
+package com.playback.models;
+
+import com.playback.standards.AudioEncoding;
+import com.playback.standards.Bitrate;
+import com.playback.standards.VideoEncoding;
+import com.playback.standards.VideoResolution;
+
+import java.util.Objects;
+
+public class VideoMetadata {
+    private final String title;
+    private final VideoEncoding videoEncoding;
+    private final VideoResolution videoResolution;
+    private final AudioEncoding audioEncoding;
```

Comment:
FYI, we split out audio and video streams, encoded using different encoders and bitrates.  This allows for more flexibility in our adaptive streaming decisions.

Reply:
This makes sense. This would result in a bunch more permutations of `VideoMetadata`